locale/en/locale.po

```diff
@@ -954,6 +954,9 @@ msgstr "About the Press"
 msgid "about.editorialPolicies"
 msgstr "Editorial Policies"
 
+msgid "about.editorialMasthead.linkToEditorialHistory"
+msgstr "View <a href=\"{$url}\">Editorial History</a> for this journal"
```

Comment:
OJS-specific language in OMP

Reply:
Moved this to lib with a simple generic text without journal/press/server... `View Editorial History`